README.md

```diff
@@ -28,8 +28,8 @@ Atla staff use this repository as their digital library.
   - Hyrax 2.3.3
 
 ### Important URL's
-- Local site: http://localhost:3000
-  - There is no "admin" link so you must go to http://localhost:3000/dashboard to access the backend
+- Local site: http://atla.test
```

Comment:
I think (based on working on other apps), that the `.test` endpoints are only for when someone is using dory. I know we do when developing, but the difference could be noted here for future developers.

re: [here](https://github.com/scientist-softserv/louisville-hyku/blob/main/Repo-README.md#important-urls) and [here](https://github.com/scientist-softserv/louisville-hyku/blob/main/Repo-README.md#dory)